cdap-cli/src/main/java/co/cask/cdap/cli/command/GetProgramStatusCommand.java

```diff
@@ -63,6 +63,13 @@ public String getPattern() {
 
   @Override
   public String getDescription() {
-    return String.format("Gets the status of %s", Fragment.of(Article.A, elementType.getName()));
+    switch (elementType) {
+      case FLOW:
+        return String.format("Gets the status of %s. Flows are deprecated as of release 5.0," +
+                             " use SparkStreaming as a replacement technology.",
```

Comment:
@sreevatsanraman sorry I didn't see this earlier. We should add a space between ``Spark`` and ``Streaming``.